### DISCUSSION TITLE
Maintenance and future of SAF format

### DISCUSSION THREAD

**Discussion by jarabroz** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#295] [General]
Dear SCIA,
I have heard about the end of the Structural Toolkit team which was responsible for development of AutoConverter and SAF. I read post on LinkedIn about the leave of Herman Oogink. 

What are the plans of SCIA for maintenance and future development of SAF format? Is there any official message prepared? It might help us to plan and prioritize our development in SAF / BIM area.

Thank in advance you for your answer.

**Comment by Jadelalam** [comment]
Hi Jarda, 

SAF remains a priority for us and we will keep expanding it. There is a new team that will drive this with Gino who will take the lead on this topic from Product Management.

BR. 
Jad

**Reply by jarabroz** [reply]
Hi Jad,

Thank you for your quick reply.  Is there any plan / schedule for it? I know guys from SCIA where in the lead of the community - taking care of community here on GitHub. writing documentation, releasing versions etc.  That was also the motivation of my question, as I saw some silence here.

Fingers crossed for the restarting of the SAF development.
